sub-packages/bionemo-llm/src/bionemo/llm/data/masking.py

```diff
@@ -36,6 +36,8 @@ class BertMaskConfig:
     mask_prob: float = 0.15
     mask_token_prob: float = 0.8
     random_token_prob: float = 0.1
+    minimum_masked_tokens: int = 1
```

Comment:
one idea: make this an Optional[int] = None, then if `mask_prob==0` default it to 0 otherwise 1, unless the user supplies something not None?